src/StreamJsonRpc/IJsonRpcInstanceContainer.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace StreamJsonRpc
+{
+    /// <summary>
+    /// Interface to contain an instance of <see cref="JsonRpc"/>.
```

Comment:
>Interface to contain an instance of [](start = 8, length = 35)

Reword for clarity for others: `Interface optionally implemented by <see cref="IJsonRpcMessageFormatter" /> implementations that need a reference to their owner <see cref="JsonRpc" /> class.`